content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,51 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - एक ऐप की डेप्लॉयमेंट
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                    पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है। प्रत्येक पॉड आपके क्लस्टर पर चल रहे कार्यभार के एक हिस्से का प्रतिनिधित्व करता है। <a href="/docs/concepts/workloads/pods/">पॉड्स के बारे में अधिक जानें</a>।
```

Comment:
```suggestion
                    पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है। प्रत्येक पॉड आपके क्लस्टर पर चल रहे कार्यभार के एक हिस्से का प्रतिनिधित्व करता है। <a href="/docs/concepts/workloads/pods/">पॉड के बारे में अधिक जानें</a>।
```

Reply:
`पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है।` the word of choice doesn't make a proper sentence, can you look into it.

Reply:
`पॉड के बारे में अधिक जानें` it seems fine for now, but we need a better wording for CTA sentences.

Reply:
how about keeping it as just `और अधिक जानें`, rather than specifying !! as it is already clear in the sentence that wr are talking about Pods?! 🤔

Reply:
> `पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है।` the word of choice doesn't make a proper sentence, can you look into it.

`पॉड कुबेरनेट्स एप्लिकेशन की मूल शुरुआत है।` does this looks fine.

Reply:
> `पॉड के बारे में अधिक जानें` it seems fine for now, but we need a better wording for CTA sentences.

`अधिक जानकारी के लिए` I've seen this in many hindi docs.